packages/platforms/dni/x86-64/x86-64-dni-wb2448/onlp/builds/src/module/src/sfpi.c

```diff
@@ -0,0 +1,160 @@
+/************************************************************
+ * <bsn.cl fy=2014 v=onl>
+ *
+ *        Copyright 2014, 2015 Big Switch Networks, Inc.
+ *        Copyright 2017 Delta Networks, Inc.
+ *
+ * Licensed under the Eclipse Public License, Version 1.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ *        http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the
+ * License.
+ *
+ * </bsn.cl>
+ ************************************************************
+ *
+ *
+ ***********************************************************/
+#include <onlp/platformi/sfpi.h>
+#include <x86_64_dni_wb2448/x86_64_dni_wb2448_config.h>
+#include "x86_64_dni_wb2448_log.h"
+
+#define SFP_PORT_BASE	49
+
+static int sfp_count__ = x86_64_dni_wb2448_CONFIG_SFP_COUNT;
```

Comment:
@jnealtowns The current method to read the SFP EEPROM is through MAC.
But it needs to initialize the system by executing BCM sdk first. 
So we won't implement it in the current schedule.